[CUBRIDMAN-254] Manual for COUNT_ONLY optimization and Enhancing group_concat() (#560)
CUBRID/cubrid-manual: en/sql/function/analysis_fn.rst

    ======================
      '35791113'

.. note::

>     **GROUP_CONCAT** performance for processing many rows has been improved starting from CUBRID 11.3 by optimizing internal string operations.

Reviewer: ```suggestion
    **GROUP_CONCAT** performance for processing large numbers of rows has been improved starting from CUBRID 11.3 through optimization of internal string operations.
```
